RecoHI/HiTracking/python/HighPtTracking_PbPb_cff.py

```diff
@@ -26,19 +26,10 @@
 from RecoHI.HiTracking.HISelectedTracks_cfi import *
 
 ### paths
-hiBasicTracking = cms.Sequence(hiPixelVertices
-                                * hiPrimSeeds
-                                * hiPrimTrackCandidates
-                                * hiGlobalPrimTracks
-                                * hiTracksWithQuality
+hiBasicTrackingTask = cms.Task(hiPixelVerticesTask
+                                , hiPrimSeedsTask
+                                , hiPrimTrackCandidates
+                                , hiGlobalPrimTracks
+                                , hiTracksWithQualityTask
                                 )
-
-
-
-
-
-
-
-
-
-
+hiBasicTracking = cms.Sequence(hiBasicTrackingTask)
```

Comment:
This sequence was only used in RecoHI/HiTracking/python/HiTracking_cff.py, and it can be removed now